web/redux/reducers/active-rooms-reducer.js

```diff
@@ -0,0 +1,17 @@
+import Immutable from 'immutable';
+
+import * as activeRoomsTypes from '../constants/active-rooms-contants';
+
+export default function activeRooms(state = Immutable.Map(), action) {
+  switch (action.type) {
+    case (activeRoomsTypes.ACTIVE_ROOMS_LOADED_MORE): {
+      return state.merge(Immutable.fromJS(action.activeRooms.reduce((obj, room) => {
```

Comment:
I changed the client side format a couple of times therefore I thought keeping the endpoint as generic as possible would be best. If you would like me to change it, I can update it.